### ISSUE TITLE
Syntax error not detected

### ISSUE THREAD

**Issue by davidkuridza** [rodjek/puppet-lint#257]
puppet-lint doesn't report any problems detected when running agains the following manifest:

``` puppet
class { '::nginx':
  http_cfg_append        => [
    'include /etc/nginx/my.d/*.conf',
  ]
  names_hash_bucket_size => '128',
}
```

``` shell
$ puppet-lint nginx.pp
$ echo $?
0
```

There is `,` missing after `]`, when running puppet, an error is thrown:

``` shell
$ puppet apply nginx.pp
Could not parse for environment production: Syntax error at 'names_hash_bucket_size'; expected '}' at nginx.pp:5 on node dev.local
$ echo $?
1
```

I'm using version `0.3.2`.
